### PR TITLE
FEAT(server): Add error code when UDP socket binding fails

### PR DESCRIPTION
**Pull request by Snowknight26**
### Checks

- [X] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

### CODE REVIEW

File: src/murmur/Server.cpp [hunk 1]
```diff
@@ -177,7 +177,7 @@ Server::Server(int snum, QObject *p) : QThread(p) {
 			}
 
 			if (::bind(sock, reinterpret_cast< sockaddr * >(&addr), len) == SOCKET_ERROR) {
-				log(QString("Failed to bind UDP Socket to %1").arg(addressToString(ss->serverAddress(), usPort)));
+				log(QString("Failed to bind UDP Socket to %1: %2").arg(addressToString(ss->serverAddress(), usPort), WSAGetLastError()));
```
Comment (davidebeatrici) [inline src/murmur/Server.cpp:180]:
```suggestion
#ifdef Q_OS_WIN
				log(QString("Failed to bind UDP Socket to %1: %2").arg(addressToString(ss->serverAddress(), usPort), WSAGetLastError()));
#else
				log(QString("Failed to bind UDP Socket to %1: %2").arg(addressToString(ss->serverAddress(), usPort), errno));
#endif
```

Comment (Snowknight26) [reply src/murmur/Server.cpp:180]: Thanks! I don't have a non-Windows environment to compile this on but I see that's needed due to the CI errors.

**Review by Krzmbrzl — APPROVED** [review]

**Comment by Krzmbrzl** [comment]
The code only needs to be clang-formatted (with exactly version 10)

**Comment by Krzmbrzl** [comment]
Thank you for this contribution :+1:

**Pull request merged**